corehq/apps/userreports/expressions/specs.py

```diff
@@ -304,3 +305,31 @@ def _get_forms(self, case_id, context):
 
         context.set_cache_value(cache_key, xforms)
         return xforms
+
+
+class SubcasesExpressionSpec(JsonObject):
+    type = TypeProperty('get_subcases')
+    case_id_expression = DefaultProperty(required=True)
+
```

Comment:
Don't have too much context on how these expressions work, but would we ever only want only one type of subcase returned here? (i.e. either only `child` or only `extension`?)